.github/workflows/upload-nextjs-integration-test-results.yml

```diff
@@ -0,0 +1,63 @@
+# Reusable workflow to upload next.js integration test result to specific branch `nextjs-integration-test-data`
```

Comment:
Is using a branch for this what we want long term? Would it be useful to make a dashboad in Tinybird or Preset or some other thing we use internally to track metrics over time?

Reply:
I think https://github.com/vercel/turbo/pull/4287#discussion_r1144030161 answers, I wish to upload this to somewhere else, like honeycomb or some sort of obsevability so we can consolidate all of the data into single place with more query-able. Uploading it into this branch was immediate stopgap following turbopack's benchmark practices to unblock us immediately.